src/utils/constants.js

```diff
@@ -0,0 +1,10 @@
+/* eslint-disable max-len */
+const supportedCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+const errors = {
+  incorectText: 'Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
```

Comment:
This part is duplicating `Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>"`
Better to create constant for it